GoogleForCreators/web-stories-wp
# Document Panel: Status Panel Redesign

## Context

Updates the status panel
- [Radio group accessibility spec](https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20161214/examples/radio/radio-1/radio-1.html)
- [Designs](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3398%3A94450)

## Summary

Updates the status panel

## Relevant Technical Choices

Decided not to add the `RadioGroup` to design system. 
- After adding the `label` and the `hint` (the text) to the `Radio`, the old `RadioGroup` didn't need to show text or style anything anymore.
- Radio buttons are generally easy to group together so having a presentational grouping component seems redundant.
- Keeping the `RadioGroup` in the editor keeps the `useRadioNavigation` specific to the editor.

Happy to add this if we think that we want it in the design system. It would be slightly easier to have a group of radio buttons already set up with proper accessibility enabled.

## To-do

- [x] determine if the 'password' radio button should be shown (Answer: not yet)

## User-facing changes

|Before|After|
|--|--|
|![old-news](https://user-images.githubusercontent.com/22185279/110711577-09506d80-81bd-11eb-8480-f5c20d3148af.gif)|![status-panel](https://user-images.githubusercontent.com/22185279/110832640-c643d800-8258-11eb-848f-cbc489182966.gif)|


## Testing Instructions

1. Open editor
2. Click document panel on right side
3. Play with radio buttons. It should be keyboard accessible 

### QA






### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #6501
Fixes #6513
